build.gradle.kts

```diff
@@ -22,7 +22,7 @@ dependencies {
     })
     implementation("org.hamcrest:hamcrest-core:1.3")
     implementation("javax.help:javahelp:2.0.05")
-//  implementation("com.fifesoft:rsyntaxtextarea:2.6.1") // Currently using 3.0.0-SNAPSHOT
+    implementation("com.fifesoft:rsyntaxtextarea:3.0.8") // Currently using 3.0.0-SNAPSHOT
```

Comment:
I suggest to remove the comment.

Reply:
Removed in following commit. Thank you, @maehne .